### PR TITLE
[GH-115] Peers sync

### LINKED ISSUE #115: getting random peer from peers we connect to
- [ ] randomize peers to get peers from
 - [ ] add new peers randomly until limit
 - [ ] maybe remove some peers randomly if at limit to get some variation

### PR DESCRIPTION
**Pull request by cytadela8**
### This PR solves following problems:
- When we know less then `peers_target_count` peers we add random peers from peers known by peers we know. (We request peers list from peers we know and add random peers from that list)
- Max known peers count is now limited. When we try to add another one, but the max count is reached, we at random: remove one of known or ignore the new one.
- When we know at least `peers_target_count` peers we remove one randomly every "sync loop".

### Stuff missing for ready peers sync:
- [ ] adding peer when unknown node connects to us. (#116)
- [ ] testing (#119)

This replaces #120 
This closes #115

### CODE REVIEW

**Comment by cytadela8** [comment]
Please check and advice about code style. This is my first contribution so I probably violated some established rules.

File: apps/aecore/lib/aecore/peers/worker.ex [hunk 1]
```diff
@@ -72,28 +75,35 @@ defmodule Aecore.Peers.Worker do
...
-        {:reply, :error, %{state | peers: peers}}
+  def handle_call({:add_peer,uri}, _from, %{peers: peers, nonce: own_nonce} = state) do
+    if Map.has_key?(peers, uri) do
+      Logger.debug(fn ->
+              "Skipped adding #{uri}, already known" end)
+      {:reply, {:error, "Peer already known"}, state}
+    else
+      case check_peer(uri, own_nonce) do
+        {:ok, info} ->
+          if map_size(peers) < @peers_max_count
```
Comment (thepiwo) [inline apps/aecore/lib/aecore/peers/worker.ex]: can we refactor this if condition into a function, to give it a name

**Comment by thepiwo** [comment]
code style is good so far, we have no specific rules yet

File: apps/aecore/lib/aecore/peers/worker.ex [hunk 2]
```diff
@@ -189,6 +188,11 @@ defmodule Aecore.Peers.Worker do
     end
   end
 
+  defp should_a_peer_be_added peers_count do
```
Comment (cytadela8) [inline apps/aecore/lib/aecore/peers/worker.ex]: @thepiwo Is this what you thought of?

Comment (thepiwo) [reply apps/aecore/lib/aecore/peers/worker.ex]: yes great!

**Pull request merged**